runtime/queries/julia/injections.scm

```diff
@@ -1,5 +1,16 @@
-((triple_string) @injection.content
- (#set! injection.language "markdown"))
+(
+  (string_literal) @injection.content
+  (#set! injection.language "markdown"))
```

Comment:
Won't this try to inject markdown into all strings?

Reply:
This was one of the few parts that I didn't change. I modified it, so that now only docstrings will be captured here.